src/shared/components/features-section/index.module.css

```diff
@@ -0,0 +1,44 @@
+@import "shared/styles/imports/variables.css";
+@import "shared/styles/imports/custom-medias.css";
+
+.container {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    background: var(--color-white);
+    color: var(--color-dark-blue);
+
+    & .content {
+        width: var(--container-width);
+        padding: 10% 0;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        text-align: center;
+
+        & h1 {
+            font-weight: bold;
+            text-align: center;
+            line-height: 48px;
+        }
+
+        & .sectionDescription {
+            display: flex;
+            justify-content: center;
+
+            & p {
+                width: 60%;
+                font-size: 15px;
```

Comment:
You should put the font sizes and weights in variables too.

Reply:
Done.